scripts/sst_prep_obs.sh

```diff
@@ -13,6 +13,27 @@ echo SST SOURCE = ${sst_source}
 
 cd $DCOM_ROOT
 
+ObsRunDir=$RUNCDATE/Data/${CDATE}    #TODO: Should not be needed here ...
```

Comment:
Is already at the main_obs_prep.sh, it is not needed here.

Reply:
I was having issues with obsrundir not defined, does it run ok for you when commented out?